install/automationservicebroker/deploy-automation-broker-apb.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: automation-broker-apb
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: automation-broker-apb
+    namespace: "${NAMESPACE}"
+
+- apiVersion: rbac.authorization.k8s.io/v1beta1
+  kind: ClusterRoleBinding
+  metadata:
+    name: automation-broker-apb
+  roleRef:
+    name: cluster-admin
```

Comment:
Thank you for making this very clear.  Do you have a link to where this was litigated before?

@sdodson do you install this with ansible?  Do you grant this permission?